### DISCUSSION TITLE
新安装的最新版本，应用商品里面是空的

### DISCUSSION THREAD

**Discussion by wbteve** [1Panel-dev/1Panel#5913] [Q&A]
### 1Panel 版本

v1.10.12-lts

### 问题描述

安装完后打开应用商店，一个应用都没有，更新也没有作用
![WX20240723-104615@2x](https://github.com/user-attachments/assets/c3ffb0a4-ddc2-4cb1-a2ae-e9d1220db7cc)

### 重现步骤

安装完后打开应用商店，一个应用都没有，更新也没有作用
![WX20240723-104615@2x](https://github.com/user-attachments/assets/c3ffb0a4-ddc2-4cb1-a2ae-e9d1220db7cc)

**Comment by zhengkunwang223** [comment]
![image](https://github.com/user-attachments/assets/55e5f663-49ec-49d3-9f56-4157d99d3936)
 看一下这里
目前我们的应用商店 可能有一些海外机器的特定节点同步不到
